packages/core/src/index.ts

```diff
@@ -1,4 +1,4 @@
-const SIDEDISH_API = 'https://api.sidedish.dev/v1';
+const SIDEDISH_API = 'https://api.sidedish.dev/v1/safe-sessions';
```

Comment:
nope. you should create another const with SafeSessionURL = `${SIDEDISH_API}/safe-sessions`